index.js

```diff
@@ -29,18 +29,6 @@ const HOME_MENU_BUTTONS_ARRAY = [
     },
 ]
 
-const ELEMENT_IDS = {
```

Comment:
You mentioned that you were going to use it in a future. I hope this is coming in some time.